tests/testthat/test-memprof.R

```diff
@@ -1,12 +1,13 @@
-test_that("with_monitor can monitor memory", {
+test_that("with_monitor can monitor system memory", {
   f <- function(msg) {
     Sys.sleep(1)
     z <- numeric(1e8) ## 800MB
     Sys.sleep(1)
     msg
   }
 
-  memprof <- with_monitor(f("hello"))
+  gc()
```

Comment:
do we not do a gc first?

Reply:
hmm actually getting some flaky behaviour which is fixed by a gc on mac. Not quite sure why...